cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
```

Comment:
need to add "import UIKit" for UIViewController and other UI related code to work